tokio/src/macros/join.rs

```diff
@@ -21,7 +21,7 @@ macro_rules! doc {
         /// The supplied futures are stored inline and do not require allocating a
         /// `Vec`.
         ///
-        /// ### Runtime characteristics
+        /// ### Fairness
```

Comment:
Note sure why this uses three hashes, but it's a sub-section of `Notes` so let's use two.
```suggestion
        /// ## Fairness
```